src/resolver/resolver.zig

```diff
@@ -1129,6 +1129,43 @@ pub const Resolver = struct {
             return .{ .not_found = {} };
         }
 
+        if (strings.startsWith(import_path, "file:///")) {
```

Comment:
`startsWith` returns true for empty strings and the right-hand side is comptime-known which means we can make it faster through more specific codegen


```suggestion
        if (strings.hasPrefixComptime(import_path, "file:///")) {
```